server/test/test.js

```diff
@@ -31,15 +31,15 @@ describe('Save and load tests', () => {
     const testText = 'Testing rest-api'
```

Comment:
## Testejä

Kaikille uusille api-routeille pitäis olla testit.

Reply:
Tehty.